scalableminds/webknossos
# Allow expanding/collapsing tree groups in locked annotations

### URL of deployed dev instance (used for testing):
- https://fixskeletongroupexpansionlockedannotations.webknossos.xyz/

### Steps to test:
- open locked annotation and expand/collapse skeleton groups
  - test "collapse/expand all subgroups" actions, at least with some nested groups
  - do the same in the segments tab
  - make sure neither expansion is actually saved
- in an editable annotation, make sure expansion/collapsing of segment and skeleton groups works like before
- make sure you can still save whether the groups are expanded or collapsed

### TODOs:
- [x] find out how to improve action. right now manipulated segment groups can be set like this, although the annotation is locked
- [x] improve naming
- [x] add the same logic for segments

### Issues:
- follow-up for #7939

------
(Please delete unneeded items, merge only when none are left open)
